ocaml/xapi/xapi_pool_update.ml

```diff
@@ -340,18 +340,31 @@ let introduce ~__context ~vdi =
 let pool_apply ~__context ~self =
   let pool_update_name = Db.Pool_update.get_name_label ~__context ~self in
   debug "pool_update.pool_apply %s" pool_update_name;
-  ()
+
+  let applied_hosts = Db.Pool_update.get_hosts ~__context ~self in
```

Comment:
Thanks @lindig 
Comments reworked. Please help to review the latest commit 1c2a96b0645bccbd1ff5ef70d28d370336216078